src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1003,6 +1003,7 @@ public static function redirect($strLocation, $intStatus=303)
 			exit;
 		}
 
+		$strLocation = ltrim($strLocation, '/');
```

Comment:
But this also means that we won't be able to detect wrong input anymore. Also, I think the `ltrim()` call should be on line 1013:

```php
// Make the location an absolute URL
if (!preg_match('@^https?://@i', $strLocation))
{
	$strLocation = \Environment::get('base') . ltrim($strLocation, '/');
}
```